tee-worker/ts-tests/vc.test.ts

```diff
@@ -26,29 +26,29 @@ describeLitentry('VC test', async (context) => {
     const aesKey = '0x22fc82db5b606998ad45099b7978b5b4f9dd4ea6017e57370ac56141caaabd12';
     var indexList: HexString[] = [];
 
-    step('check user sidechain storage before create', async function () {
-        const resp_shieldingKey = await checkUserShieldingKeys(
-            context,
-            'IdentityManagement',
-            'UserShieldingKeys',
-            u8aToHex(context.substrateWallet.alice.addressRaw)
-        );
-        assert.equal(resp_shieldingKey, '0x', 'resp_shieldingKey should be empty before set');
-    });
+    // step('check user sidechain storage before create', async function () {
+    //     const resp_shieldingKey = await checkUserShieldingKeys(
+    //         context,
+    //         'IdentityManagement',
+    //         'UserShieldingKeys',
+    //         u8aToHex(context.substrateWallet.alice.addressRaw)
+    //     );
+    //     assert.equal(resp_shieldingKey, '0x', 'resp_shieldingKey should be empty before set');
+    // });
```

Comment:
What's the reason for commenting out these? If it's needed we should have the comment somewhere.

Reply:
Accidentally commented out for local testing, it will be restored